lib_src/core/inspect_bits.ex

```diff
@@ -0,0 +1,116 @@
+defmodule Toolshed.Core.InspectBits do
+  @type conversion :: %{
+          :binary => binary(),
+          :decimal => binary(),
+          :hex => binary(),
+          :octal => binary()
+        }
+
+  @doc """
+  Prints out information on the given numeric value.
+  """
+  @spec inspect_bits(integer | binary) :: :"do not show this result in output"
+  def inspect_bits(number) do
+    conversion = convert_bits(number)
+
+    [
+      [:cyan, "Decimal    ", :reset, " : ", conversion.decimal],
+      [:cyan, "Hexadecimal", :reset, " : ", conversion.hex],
+      [:cyan, "Octal      ", :reset, " : ", conversion.octal],
+      [:cyan, "Binary     ", :reset, " : ", format_base2_string(conversion.binary)]
+    ]
+    |> Enum.map(&IO.ANSI.format(&1))
+    |> Enum.intersperse('\n')
+    |> IO.puts()
```

Comment:
I like the colorization for this. It makes the numbers really easy to read.

Here's a simplification:

```suggestion
    [
      [:cyan, "Decimal    ", :reset, " : ", conversion.decimal, '\n'],
      [:cyan, "Hexadecimal", :reset, " : ", conversion.hex, '\n'],
      [:cyan, "Octal      ", :reset, " : ", conversion.octal, '\n'],
      [:cyan, "Binary     ", :reset, " : ", format_base2_string(conversion.binary)]
    ]
    |> IO.ANSI.format()
    |> IO.puts()
```

`IO.ANSI.format/1` can handle embedded lists (since they're `ansidata`). Also, I liked the manual addition of `\n` rather than `Enum.intersperse/2` since once the `Enum.map/2` was removed, removing the other `Enum` call made this a simple format/puts process.